### PR TITLE
Dialog request hierarchie

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** The dialogs was in old design.

**Solution:** Update to new design.

Captura de tela de 2019-03-11 10-51-20
Captura de tela de 2019-03-11 10-51-33
Captura de tela de 2019-03-11 10-51-20
Captura de tela de 2019-03-11 16-46-13

user-images.githubusercontent.com/18709274/54130397-0fa18880-43ef-11e9-873a-1a52929696e3.png)

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/requests/analyseHierarchyRequestController.js [hunk 1]
```diff
@@ -27,6 +28,15 @@
...
         })();
 
+        analyseHierReqCtrl.showButtonAccept = function(){
+            return !analyseHierReqCtrl.hasToRemoveLink || 
+                analyseHierReqCtrl.hasToRemoveLink && analyseHierReqCtrl.analyseInstitutions;
+        }
+
+        analyseHierReqCtrl.isDescriptionRemoveLink = function(){
+            return analyseHierReqCtrl.hasToRemoveLink && !analyseHierReqCtrl.analyseInstitutions;
+        }
```
Comment (ruanres) [inline frontend/requests/analyseHierarchyRequestController.js]: ;

File: frontend/requests/analyseHierarchyRequestController.js [hunk 2]
```diff
@@ -27,6 +28,15 @@
             analyseHierReqCtrl.child = child;
         })();
 
+        analyseHierReqCtrl.showButtonAccept = function(){
+            return !analyseHierReqCtrl.hasToRemoveLink || 
+                analyseHierReqCtrl.hasToRemoveLink && analyseHierReqCtrl.analyseInstitutions;
+        }
```
Comment (ruanres) [inline frontend/requests/analyseHierarchyRequestController.js]: ;

File: frontend/requests/analyse_hierarchy_request_dialog.html [hunk 3]
```diff
@@ -1,88 +1,82 @@
...
+<md-dialog class="analyse_request__dialog" flex-sm="80" flex-xs="90" flex-gt-sm="50">
     <md-dialog-content class="md-dialog-content" layout-gt-sm="column" style="padding: 1em;"
     ng-if="!analyseHierReqCtrl.showProcessingMessage()">
         <div ng-if="!analyseHierReqCtrl.hasToRemoveLink">
             <h3> Confirmar vínculo </h3>
-            <p>
+            <p class="analyse_request__description">
                 Para aceitar o vínculo e tornar <b>{{ analyseHierReqCtrl.child.name }}</b> uma subornadinada de
-                <b>{{ analyseHierReqCtrl.parent.name }}</b>, clique em Confirmar. Caso queira rejeitar, clique em cancelar.
+                <b>{{ analyseHierReqCtrl.parent.name }}</b>. Clique em aceitar, caso não queira se vincular a esta instituição clique em rejeitar.
```
Comment (ruanres) [inline frontend/requests/analyse_hierarchy_request_dialog.html]: </b>, clique em aceitar. Caso não queira ...

File: frontend/requests/analyseHierarchyRequestController.js [hunk 4]
```diff
@@ -27,6 +28,15 @@
...
+
+        analyseHierReqCtrl.isDescriptionRemoveLink = function(){
```
Comment (ruanres) [inline frontend/requests/analyseHierarchyRequestController.js]: the name of this function is strange

File: frontend/requests/analyse_hierarchy_request.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.analyse_request__description{
```
Comment (ruanres) [inline frontend/requests/analyse_hierarchy_request.css]: the correct pattern is analyse-request__description. See those [examples](https://seesparkbox.com/foundry/bem_by_example).

File: frontend/requests/analyse_hierarchy_request_dialog.html [hunk 6]
```diff
@@ -1,88 +1,82 @@
...
-            <div layout="row" md-colors="{background: 'default-grey-200'}" layout-margin flex="95" style="margin: 0;">
-                <div>
-                <img style="width: 75px; height: 75px; border-radius: 50%;" ng-src="{{ analyseHierReqCtrl.child.photo_url || 'app/images/avatar.png'}}"
-                    alt="{{ analyseHierReqCtrl.child.name }}"></img>
-                </div>
-                <div layout="column" layout-align="center start">
-                <b>{{ analyseHierReqCtrl.child.name || analyseHierReqCtrl.child.sender_name}}</b>
-                <span>{{ analyseHierReqCtrl.child.institutional_email }}</span>
-                <span>{{ analyseHierReqCtrl.child.office }}</span>
+            <div layout="column">
```
Comment (ruanres) [inline frontend/requests/analyse_hierarchy_request_dialog.html:59]: why are you still using layout column

**Review by ruanres — CHANGES_REQUESTED** [review]

File: frontend/requests/analyseHierarchyRequestController.js [hunk 7]
```diff
@@ -27,6 +28,15 @@
             analyseHierReqCtrl.child = child;
         })();
 
+        analyseHierReqCtrl.showButtonAccept = function(){
```
Comment (JuliePessoa) [inline frontend/requests/analyseHierarchyRequestController.js:35]: jsdoc

File: frontend/requests/analyseHierarchyRequestController.js (hunk 4 above)
Comment (JuliePessoa) [inline frontend/requests/analyseHierarchyRequestController.js]: jsdoc

File: frontend/requests/analyse_hierarchy_request_dialog.html [hunk 8]
```diff
@@ -1,88 +1,82 @@
...
                 <div layout="row">
-                <div flex="5" layout="row" layout-align="center">
-                    <div style="width: 8px;" md-colors="{background: 'default-teal-500'}"></div>
-                </div>
-                <div layout="row" md-colors="{background: 'default-grey-200'}" layout-margin flex="95" style="margin: 0;">
-                    <div>
-                    <img style="width: 75px; height: 75px; border-radius: 50%;" ng-src="{{ analyseHierReqCtrl.child.photo_url || 'app/images/avatar.png'}}"
-                        alt="{{ analyseHierReqCtrl.child.name }}"></img>
+                    <div flex="5" layout="row" layout-align="center">
+                        <div style="width: 8px;" md-colors="{background: 'default-teal-500'}"></div>
```
Comment (JuliePessoa) [inline frontend/requests/analyse_hierarchy_request_dialog.html]: move style

Comment (JuliePessoa) [inline frontend/requests/analyse_hierarchy_request_dialog.html]: what is this div?

File: frontend/requests/analyse_hierarchy_request_dialog.html [hunk 9]
```diff
@@ -1,88 +1,82 @@
-<md-dialog flex-sm="80" flex-xs="100" flex-gt-sm="50">
+<md-dialog class="analyse_request__dialog" flex-sm="80" flex-xs="90" flex-gt-sm="50">
```
Comment (JuliePessoa) [inline frontend/requests/analyse_hierarchy_request_dialog.html]: remember that on desktop screen size, it should remain the same thing.

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/requests/analyse_hierarchy_request_dialog.html (hunk 8 above)
Comment (MaianaB) [reply frontend/requests/analyse_hierarchy_request_dialog.html]: its the green bar

File: frontend/requests/analyse_hierarchy_request_dialog.html (hunk 9 above)
Comment (MaianaB) [reply frontend/requests/analyse_hierarchy_request_dialog.html]: added max-height

**Review by ruanres — APPROVED** [review]

**Review by Luiz-FS — APPROVED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Pull request merged**